### ISSUE TITLE
Alert user to use of Amazon affiliate link tags by default

### ISSUE THREAD

**Issue by sebj** [laconalabs/LaconaApp#122]
I've only just checked Lacona's Web Search settings and noticed it appears to use an Amazon affiliate tag on the end of all Amazon searches (`http://www.amazon.com/s?url=... ...&tag=lacona-20`).

I realise this can be removed, but it seems pretty sneaky and many users might not notice or change it. It'd be nice to at least have a small text label letting the user know money is being made from their Amazon searches, or even a checkbox to "Support Lacona" by allowing affiliate tags.

**Comment by brandonhorst** [comment]
Yeah - I was actually planning to add that setting when I introduced payment, but neglected to do so. I will make sure that I at least add a note to the page.
